main/src/hadd-argparse.py

```diff
@@ -33,6 +33,8 @@ def get_argparse():
 	parser.add_argument("-ff", help="The compression level use is the one specified in the first input")
 	parser.add_argument("-f0", help="Do not compress the target file")
 	parser.add_argument("-f6", help="Use compression level 6. (See TFile::SetCompressionSettings for the support range of value.)")
+	parser.add_argument("-objects", help="List of space-separated keys (folders, histograms) to either include or exclude from the input files")
```

Comment:
I don't like much having space-separated keys in this context (it makes the parsing code harder than it could be and in general it intermingles with the space-separated top-level options), maybe it'd be better to choose a different separator like `,` or `;`.
However see my other comment about rebasing your PR first